packages/workbox-range-requests/src/utils/calculateEffectiveBoundaries.ts

```diff
@@ -44,14 +44,15 @@ function calculateEffectiveBoundaries(
   let effectiveStart: number;
   let effectiveEnd: number;
 
-  if (start && end) {
+  // != and == are used so both null and undefined are treated the same.
```

Comment:
If there's a more TypeScript-y way of doing this, let me know.

Reply:
From my reading of [the code for the `parseRangeHeader()` function](https://github.com/GoogleChrome/workbox/blob/bfd5c718a81af2add2801883de1b18a85d072529/packages/workbox-range-requests/src/utils/parseRangeHeader.ts#L50-L51), only `undefined` can be returned, so I don't think it's necessary to account for `null`.

Reply:
Okay, yeah, the 4.3.1 version returned `null` in that scenario, so I thought without checking to closely that `null` was still possible.